packages/react-components/src/accordion/accordion.tsx

```diff
@@ -49,6 +49,11 @@ const AccordionItem = styled(AccordionPrimitive.Item, {
   '&:focus-within': {
     position: 'relative',
     zIndex: 1
+  },
+  '&[data-disabled]': {
+    background: '$white900',
```

Comment:
@tmbalagan background property can be avoided (opacity alone will be enough) here as the accordion component is expected to work fine in all sorts of backgrounds. Adding white background here will break the accordion ui in screens which have any other color as the background.